server/constants.py

```diff
@@ -85,46 +85,50 @@ class ExtraFileNames:
     """
     # Name for the tale config file
     tale_config = 'tale.yml'
+    license = 'LICENSE'
```

Comment:
`license` is a reserved name in Python. This might work but what do you think about changing it to `license_filename` or something?